core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -85,6 +85,10 @@ static function (string $v): int {
                         ->thenInvalid('%s')
                     ->end()
                 ->end()
+                ->scalarNode('csrf_cookie_prefix')
```

Comment:
Does it make sense to have yet another config? Is there a reason we cannot know the exact name of the cookie? Not sure if the token name should not match the cookie name? Not even sure if that's possible?

Reply:
The cookie name is currently `csrf_contao_csrf_token` because the token cookie listener is designed to handle all cookies that start with the prefix `csrf_` and `contao_csrf_token` is the name of the token.
I made it configurable now so that we don’t have to rely on the default values from `CsrfTokenCookieListener` and `RequestTokenListener` being the same. Don’t know what the best practice here is.

Reply:
Fine for me :)